RELEASE.md

```diff
@@ -8,15 +8,33 @@
 
 ## Migration guide from Kedro 0.18.* to 0.19.*
 
-# Upcoming Release 0.18.6
+# Upcoming Release 0.18.7
 
 ## Major features and improvements
 
 ## Bug fixes and other changes
-* Fixed bug that didn't allow to read or write datasets with `s3a` or `s3n` filepaths
 
 ## Breaking changes to the API
 
+# Release 0.18.6
+
+## Bug fixes and other changes
+* Fixed bug that didn't allow to read or write datasets with `s3a` or `s3n` filepaths
+* Fixed bug with overriding nested parameters using the `--params` flag
+* Fixed bug that made session store incompatible with `Kedro-Viz` experiment tracking
+
+## Migration guide from Kedro 0.18.5 to 0.18.6
+A regression introduced in Kedro version `0.18.5` caused the `Kedro-Viz` console to fail to show experiment tracking correctly. If you experienced this issue, you will need to:
+* upgrade to Kedro version `0.18.6`
+* delete any erroneous session entries created with Kedro 0.18.5 from your session_store.db stored at `<project-path>/data/session_store.db`.
+
+Thanks to Kedroids tomohiko kato, [tsanikgr](https://github.com/tsanikgr) and [maddataanalyst](https://github.com/maddataanalyst) for very detailed reports about the bug.
+
+## Upcoming deprecations for Kedro 0.19.0
+* `project_version` will be deprecated in `pyproject.toml` please use `kedro_init_version` instead.
+* Deprecated `kedro run` flags `--node`, `--tag`, and `--load-version` in favour of `--nodes`, `--tags`, and `--load-versions`.
```

Comment:
These are in 0.18.5 too, is the repetition intentional?

Reply:
I think @stichbury might have added this but I also don't think we need this - should I remove it?

Reply:
I'm of the opposite mind - I don't think we should remove it until 0.19.0 is out as the to-be-deprecated features are still there and there is still room for new users to get exposed to those features pre-0.19.0.

Reply:
We haven't been copying over all the deprecated things from the previous 0.18.x releases so far though